src/idl_parser.cpp

```diff
@@ -2499,7 +2499,7 @@ bool Parser::SupportsAdvancedUnionFeatures() const {
   return (opts.lang_to_generate &
```

Comment:
can you add a comment saying Advanced Union Features means

1. vectors of unions
2. strings in unions
3. structs in unions

As implemented, all three need to be supported

Reply:
Note: I added the comment on the second commit, even though Github does not mark this reference as outdated.